### ISSUE TITLE
Handle mounting of APFS images that have a EULA

### ISSUE THREAD

**Issue by crystalllized** [autopkg/autopkg#585]
**Describe the problem**
APFS disk images with a EULA are not able to be mounted by Autopkg

**AutoPkg output**
Output from @homebysix's test because it's easier to copy paste :)
```
Processing ~/Developer/_autopkg_org/Yohan460-recipes/JAMFConnect/JamfConnectSync.pkg.recipe...
WARNING: ~/Developer/_autopkg_org/Yohan460-recipes/JAMFConnect/JamfConnectSync.pkg.recipe is missing trust info and FAIL_RECIPES_WITHOUT_TRUST_INFO is not set. Proceeding...
URLDownloader
URLDownloader: Storing new Last-Modified header: Tue, 19 Nov 2019 14:06:09 GMT
URLDownloader: Storing new ETag header: "4f371ad8c65fb5e2955af6a12125c5f6-3"
URLDownloader: Downloaded ~/Library/AutoPkg/Cache/com.github.yohan460-recipes.pkg.JamfConnectSync/downloads/Jamf Connect Sync.dmg
EndOfCheckPhase
CodeSignatureVerifier
CodeSignatureVerifier: hdiutil imageinfo error nx_kernel_mount:1473: : checkpoint search: largest xid 4, best xid 4 @ 7
nx_kernel_mount:1473: : checkpoint search: largest xid 4, best xid 4 @ 7
nx_kernel_mount:1473: : checkpoint search: largest xid 4, best xid 4 @ 7
nx_kernel_mount:1473: : checkpoint search: largest xid 4, best xid 4 @ 7
nx_kernel_mount:1473: : checkpoint search: largest xid 4, best xid 4 @ 7
nx_kernel_mount:1473: : checkpoint search: largest xid 4, best xid 4 @ 7
nx_kernel_mount:1473: : checkpoint search: largest xid 4, best xid 4 @ 7
nx_kernel_mount:1473: : checkpoint search: largest xid 4, best xid 4 @ 7
 with image ~/Library/AutoPkg/Cache/com.github.yohan460-recipes.pkg.JamfConnectSync/downloads/Jamf Connect Sync.dmg.
~/Library/AutoPkg/Cache/com.github.yohan460-recipes.pkg.JamfConnectSync/downloads/Jamf Connect Sync.dmg is not mounted
Failed.
Receipt written to ~/Library/AutoPkg/Cache/com.github.yohan460-recipes.pkg.JamfConnectSync/receipts/JamfConnectSync.pkg-receipt-20191121-100139.plist
The following recipes failed:
    ~/Developer/_autopkg_org/Yohan460-recipes/JAMFConnect/JamfConnectSync.pkg.recipe
        Error in com.github.yohan460-recipes.pkg.JamfConnectSync: Processor: CodeSignatureVerifier: Error: ~/Library/AutoPkg/Cache/com.github.yohan460-recipes.pkg.JamfConnectSync/downloads/Jamf Connect Sync.dmg is not mounted
The following new items were downloaded:
    Download Path                                                                                                      
    -------------                                                                                                      
    ~/Library/AutoPkg/Cache/com.github.yohan460-recipes.pkg.JamfConnectSync/downloads/Jamf Connect Sync.dmg  

```
**Expected behavior**
Expected, the DMG gets mounted and operations performed. 

**Version (please complete the following information):**
 - OS version: 10.14.6
- AutoPkg Version: 1/.3.1

**Comment by homebysix** [comment]
FWIW, the `imageinfo` plist for the disk image in question _does_ appear to have the expected Properties:Software License Agreement value set to `<true/>`.

https://gist.github.com/homebysix/69cc40c202727ce414def26452a80684

**Comment by homebysix** [comment]
Both this:

```
hdiutil attach -plist -mountrandom /private/tmp -nobrowse ~/Library/AutoPkg/Cache/com.github.yohan460-recipes.pkg.JamfConnectSync/downloads/Jamf\ Connect\ Sync.dmg -plist <<< 'Y\n'
```

and this:

```
echo 'Y\n' | hdiutil attach -plist -mountrandom /private/tmp -nobrowse ~/Library/AutoPkg/Cache/com.github.yohan460-recipes.pkg.JamfConnectSync/downloads/Jamf\ Connect\ Sync.dmg -plist
```

appear to display the EULA interactively, requiring a `q` to dismiss before the `Agree Y/N?` prompt appears.

**Comment by gregneagle** [comment]
Fixed in #586
